release-notes/2023-10/README.md

```diff
@@ -0,0 +1,231 @@
+# SystemLink Enterprise 2023-10 Release Notes
+
+The 2023-10 release for SystemLink Enterprise has been published to <https://downloads.artifacts.ni.com>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+
+## New Features and Behavior changes
+
+- You can view the current and historical values of tags in dashboards.
+- Data tables have improved reliability and scalability and can support thousands of concurrent writers.
+- Systems data can be visualized in dashboards.
+- You can change the version of a package installed on a managed system.
+- You can view all tracked assets on the Assets page
+- You can add comments to a test result
+- You can connect to external MongoDB instances.
+    - This is a breaking change. Refer to **Helm Chart Breaking Changes** for detail on how to opt-out of this capability or migrate to a new MongoDB instance.
+- New Test Analytics privilege category is available.
+    - The Test Analytics privilege category has been added, and includes the Query Measurements privilege. This privilege is not currently functional and is being added in support of features that will release in a future version.
+
+## Upgrading from the release 2023-09 to the release 2023-10
+
+- This release upgrade Redis from 7.0 to 7.2. This is a breaking change. It is necessary to upgrade the Redis cluster. Helm will not perform this upgrade automatically.
+- Option #1
```

Comment:
Why would users choose one option over the other?

Reply:
@JD-Robertson Any thoughts on this? Do we need to add reasoning/context for users to pick one option over another?

Reply:
These should not be presented as two options. We should recommend the first option. If that fails, the user will need to reset the redis deployment as described in Option 2 and in the separate how to document in this PR.

Reply:
Thanks @JD-Robertson . I updated the section based on your inputs